CHANGELOG.txt

```diff
@@ -1,3 +1,5 @@
+2.14.13:
+    - add resumable on operations.
```

Comment:
@ahmadiesa-abu Update the message to be like `Add support for resumable actions`